internal/controller/etcd/reconcile_spec.go

```diff
@@ -99,6 +102,31 @@ func (r *Reconciler) syncEtcdResources(ctx component.OperatorContext, etcdObjKey
 	return ctrlutils.ContinueReconcile()
 }
 
+// cleanupEtcdResources cleans up the resources that are no longer required for the Etcd cluster.
```

Comment:
can you add a little more context in doc string? Now we can understand the context but a few months later this will be lost. Why are these resources no longer required needs to be explicitly mentioned.

Reply:
Done